services/movie.py

```diff
@@ -0,0 +1,37 @@
+from db.models import Movie
+
+
+def get_movies(genres_ids: list = None, actors_ids: list = None) -> None:

```

Comment:
The return type annotation for `get_movies` is missing. It should be `-> QuerySet[Movie]` to indicate that the function returns a QuerySet of Movie objects.